tests/commands/run_test.py

```diff
@@ -286,14 +286,14 @@ def test_global_files(cap_out, store, in_git_dir):
     ('t1', 't2', 'expected'),
     (
         (1.234, 2., b'\n- duration: 0.77s\n'),
-        (1., 1., b'\n- duration: 0s\n'),
+        (1., 1., b'\n- duration: 0.00s\n'),
```

Comment:
I prefer the previous display here

Reply:
For 0s I get it. But if you got 5.00s, it would be reported as 5s, which it wouldn't clear is faster than another measurement of 5.49s.

Reply:
that's fine, I don't want that part of this change though

Reply:
Okay I’ve undone the fixed point.